app/src/androidTest/java/org/simple/clinic/patient/PatientRepositoryAndroidTest.kt

```diff
@@ -1159,16 +1117,16 @@ class PatientRepositoryAndroidTest {
         recordedAt = recordedAt
     )
     database.bloodPressureDao().save(listOf(bpMeasurement))
-    return patientProfile.patient.toRecentPatient()
+    return patientProfile.patient.toRecentPatient(bpMeasurement)
   }
 
-  private fun Patient.toRecentPatient() = RecentPatient(
+  private fun Patient.toRecentPatient(bpMeasurement: BloodPressureMeasurement) = RecentPatient(
       uuid = uuid,
       fullName = fullName,
       gender = gender,
       dateOfBirth = dateOfBirth,
       age = age,
-      updatedAt = updatedAt
+      updatedAt = bpMeasurement.recordedAt
```

Comment:
Please update/add new tests to include `recordedAt` since that is what you consider for recent patients.

Reply:
done